src/tasks/installers/helm.ts

```diff
@@ -34,7 +34,9 @@ export class HelmTasks {
    * Returns list of tasks which perform preflight platform checks.
    */
   startTasks(flags: any, command: Command): Listr {
-    command.warn('You can also use features rich \'OLM\' installer to deploy Eclipse Che.')
+    if (isStableVersion(flags)) {
+      command.warn('Consider using the more reliable \'OLM\' installer when deploying a stable release of Eclipse Che (--installer=olm).')
```

Comment:
I don't see why OLM is
 > more reliable

Reply:
- no cluster admin rights
- OpenShift manages installation

Reply:
Maybe then `easier to use`?

Reply:
Not sure, see the related issue.